Dev config

zcorrea357 opened the pull request:
estruturando o repositorio

RodrigoMachado9 reviewed (changes requested):
criar pull request com a branch **develop!**

darkcode357 commented:
mudança efetuada com sucesso 
![image](https://user-images.githubusercontent.com/24551924/116028740-d6a3ec80-a625-11eb-972e-5315d1dc5a9c.png)

thgdsadsadsasd committed:
organização final do reposititorio

thgdsadsadsasd committed:
seme estrutura de documentação adicionada

thgdsadsadsasd committed:
# Creates a ruby-style comment
# Creates a javascript-style comment
# Creates a javascript-style comment
# add fist caracter on line

thgdsadsadsasd committed:
suport buffer.py:
- [x] len -> count
- [x] contains -> verific contain item
- [x] index -> index item
- [x] totalize -> add buffer to buffer
- [x] get -> get bytes for buffer
suporte config.py
- [x] register config future
suporte constante.py in util
- [x] KB
- [x] MB
- [x] GB
- [x] KiB
- [x] MiB
- [x] GiB
suporte log: future
- [x] Convert the length of a bytestream to human readable frm.
suporte Text mode
- [x] raw_string_into_a_python_buffer
- [x] create_separated_list_of_numbers
- [x] Creates a ruby-style comment
- [x] Creates a javascript-style comment
- [x] Creates a javascript-style comment
- [x] add fist caracter on line
suporte version
- [x] 0.0.01dev
suporte web
- [x] donwloadfile from url

RodrigoMachado9 committed:
Delete LICENSE-thglib.txt

arquivo não utilizado removido.
